README.md

```diff
@@ -245,14 +245,16 @@ If you need to use the tRPC client in SvelteKit's `load()` function for SSR, mak
 import { browser } from '$app/env';
 import type { Router } from '$lib/trpcServer';
 import * as trpc from '@trpc/client';
+import type { LoadEvent } from "@sveltejs/kit";
 
 const url = browser ? '/trpc' : 'http://localhost:3000/trpc';
-export default (loadFetch?: typeof fetch) =>
+export default (loadFetch?: LoadEvent['fetch']) =>
   trpc.createTRPCClient<Router>({
     url: loadFetch ? '/trpc' : url,
     transformer: trpcTransformer,
     ...(loadFetch && { fetch: loadFetch })
```

Comment:
As I mentioned in #18, this line has to be also changed to include a typecast:

```ts
...(loadFetch && { fetch: loadFetch as typeof fetch })
```

Otherwise it gives a type error, since `createTRPCClient` expects the Node fetch type:

![image](https://user-images.githubusercontent.com/46423131/181596529-58c51728-90b8-4572-8c6d-934d59b87ac4.png)